src/index.js

```diff
@@ -0,0 +1,131 @@
+// ------------- Wave 2 ----------------------
+const state = {
+  temp: 60,
+  city: 'Seattle',
+  sky: '☁️',
+};
+// 1. increase temp
+const addDegree = (event) => {
+  state.temp += 1;
+  console.log('state temp', state.temp);
+  const tempContainer = document.querySelector('#degrees');
+  tempContainer.textContent = state.temp;
+  updateColorsAndEmojis();
+};
+// 2. decrease temp
+const subtractDegree = (event) => {
+  state.temp -= 1;
+  console.log('state temp', state.temp);
+  const tempContainer = document.querySelector('#degrees');
+  tempContainer.textContent = state.temp;
+  updateColorsAndEmojis();
+};
+// 2. temp ranges
+const updateColorsAndEmojis = () => {
+  if (state.temp > 80) {
+    document.getElementById('degrees').style.color = 'red';
+    document.getElementById('emojis').innerText = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  } else if (state.temp >= 70 && state.temp <= 79) {
+    document.getElementById('degrees').style.color = 'orange';
+    document.getElementById('emojis').innerText = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (state.temp >= 60 && state.temp <= 69) {
+    document.getElementById('degrees').style.color = 'yellow';
+    document.getElementById('emojis').innerText = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else if (state.temp > 50 && state.temp <= 59) {
+    document.getElementById('degrees').style.color = 'green';
+    document.getElementById('emojis').innerText = '🌲🌲';
+  } else if (state.temp < 59) {
+    document.getElementById('degrees').style.color = 'teal';
+    document.getElementById('emojis').innerText =
+      '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  }
```

Comment:
The last `else if` can just be an `else`. Also, can you think of a way to rewrite this to make it so you don't need to have so many `getElementById`s repeated?